src/main/java/com/microsoft/jenkins/appservice/WebAppDeploymentRecorder.java

```diff
@@ -360,6 +359,9 @@ public FormValidation doCheckFilePath(
                 @QueryParameter String azureCredentialsId,
                 @QueryParameter String resourceGroup,
                 @QueryParameter String appName) {
+            if (StringUtils.isBlank(azureCredentialsId)) {
+                return FormValidation.ok();
```

Comment:
isblank is ok?

Reply:
When a user creates this command in the UI for the first time, the credentialId will be blank. Just ignore this parameter check for this case. The implementation here need credentailId to fetch web app information.